services/worker/README.md

```diff
@@ -102,13 +102,15 @@ Set environment variables to configure the `duckdb-index` worker (`DUCKDB_INDEX_
 Set environment variables to configure the `descriptive-statistics` worker (`DESCRIPTIVE_STATISTICS_` prefix):
 
 - `DESCRIPTIVE_STATISTICS_CACHE_DIRECTORY`: directory to which a dataset in parquet format is downloaded. Defaults to empty.
+- `DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES`: size in bytes by which the number of first `n` parquet files to compute statistics over is determined, satisfies the following condition: `sum(parquet_files[0] + ... + parquet_files[n-1]) <= DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES & sum(parquet_files[0] + ... + parquet_files[n]) > DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES`. Defaults to `100_000_000`.
```

Comment:
here too: DATASET -> SPLIT? to make it clear that it's the size of a split's data?